docs/tutorials/tqdm_progress_bar.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
<p>Instead of merging this .png could you just display the URL using markdown?</p><p> </p><p><code>&nbsp;&nbsp;"![fig](<a href="https://raw.githubusercontent.com/tensorflow/addons/59961669a0e21eb4c045d4ad38d008a529d566c2/docs/tutorials/assets/tqdm_progress_bar_demo.png" target="_blank">https://raw.githubusercontent.com/tensorflow/addons/59961669a0e21eb4c045d4ad38d008a529d566c2/docs/tutorials/assets/tqdm_progress_bar_demo.png</a>)"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/610/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
got it!

Reply:
updated in latest patch!